parts/kuberneteskubelet.service

```diff
@@ -38,15 +38,13 @@ ExecStart=/usr/bin/docker run \
         --address=0.0.0.0 \
         --allow-privileged=true \
         --enable-server \
-        --enable-debugging-handlers \
         --pod-manifest-path=/etc/kubernetes/manifests \
         --cluster-dns=${KUBELET_CLUSTER_DNS} \
         --cluster-domain=cluster.local \
         --node-labels="${KUBELET_NODE_LABELS}" \
         --cloud-provider=azure \
         --cloud-config=/etc/kubernetes/azure.json \
         --azure-container-registry-config=/etc/kubernetes/azure.json \
-        --hairpin-mode=promiscuous-bridge \
```

Comment:
verified that (according to official docs) this is already the default setting for `1.5`, `1.6`, and `1.7`

Reply:
👍 this line removal too!